src/components/inventoryList/inventoryListHelpers.js

```diff
@@ -49,6 +50,18 @@ const applySortFilters = ({ filter = {}, onSort, query = {} }) => {
   return updatedFilter;
 };
 
+const applyWrappableFilters = ({ filter = {} }) => {
+  const updatedFilter = { ...filter };
+
+  if (Array.isArray(updatedFilter.transforms)) {
+    updatedFilter.transforms = updatedFilter.transforms.push(wrappable);
```

Comment:
required:
this should just be 
```
updatedFilter.transforms.push(wrappable);
```
remove the `updatedFilter.transforms =` otherwise it'll result in transform being equal to the array length